github/Organization.py

```diff
@@ -291,6 +291,18 @@ def add_to_public_members(self, public_member):
             self.url + "/public_members/" + public_member._identity
         )
 
+    def add_member(self, member):
```

Comment:
I think the name should be `add_membership` to be consistent with the [`Team` API](https://github.com/PyGithub/PyGithub/blob/cbfe8d0f623ca29d984ec09d2b566e9ab10ae024/github/Team.py#L128).